docs/sophia_stdlib.md

```diff
@@ -12,40 +12,233 @@ in the scope and do not need any actions to be used, while the others require so
 
 The out-of-the-box namespaces are:
 
+- [Address](#address)
+- [AENS](#aens)
+- [Auth](#auth)
 - [Bits](#bits)
 - [Bytes](#bytes)
+- [Call](#call)
+- [Chain](#chain)
 - [Char](#char)
+- [Contract](#contract)
+- [Crypto](#crypto)
 - [Int](#int)
 - [Map](#map)
-- [Address](#address)
-- [Crypto](#crypto)
-- [Auth](#auth)
 - [Oracle](#oracle)
-- [AENS](#aens)
-- [Contract](#contract)
-- [Call](#call)
-- [Chain](#chain)
 
 The following ones need to be included as regular files with `.aes` suffix, for example
 ```
 include "List.aes"
 ```
 
-- [List](#list)
-- [Option](#option)
-- [String](#string)
-- [Func](#func)
-- [Pair](#pair)
-- [Triple](#triple)
 - [Bitwise](#bitwise)
 - [BLS12_381](bls12_381)
+- [Func](#func)
 - [Frac](#frac)
+- [List](#list)
+- [Option](#option)
+- [Pair](#pair)
 - [Set](#set-stdlib)
```

Comment:
I see that you have deleted `set-stdlib` name from below. Are you sure that clicking this will direct you to the right place?

Reply:
good point, I re-added this as we have other sub-headings in the `Bits` section where `set` is already present. I fixed it now and tested locally